statsmodels/stats/contingency_tables.py

```diff
@@ -370,13 +353,27 @@ def local_log_oddsratios(self):
 
     @cache_readonly
     def local_oddsratios(self):
-        # docstring for cached attributes in init above
+        """
+        Local odds ratios calculated from each 2x2 subtable formed from
+        adjacent rows and columns.
+        """
 
         return np.exp(self.local_log_oddsratios)
 
     @cache_readonly
     def cumulative_log_oddsratios(self):
-        # docstring for cached attributes in init above
+        """
+        Cumulative log odds ratio at a pair of thresholds
+
+        Notes
+        -----
+        The cumulative log odds ratio at a given pair of thresholds is
+        calculated by reducing the table to a 2x2 table based on
+        dichotomizing the rows and columns at the given thresholds.
```

Comment:
Below uses the wording "cutting  the rows and columns at a given point".  Assuming this has the same meaning, I'd suggest the same wording.

Reply:
I know what dichotomizing means but not what happens if you cut rows or columns.
I also don't understand why R and pandas are cutting instead of discretizing or binning or ...

Reply:
This is just a copy/paste from the Attributes section in master. I only did minimal reformatting so that it wasn't toally incoherent.